### DISCUSSION TITLE
Spanish RimWorld Steam Store Translation

### DISCUSSION THREAD

**Discussion by TiaPixel** [Ludeon/RimWorld-Spanish#74] [General]
Hello, I'm Tia! I'm helping Ludeon Studios with translating the Royalty and Ideology Steam store pages. Would anyone from the GitHub Spanish team be interested in helping me translate the store pages from English to Spanish? :)

Please respond with a way I can contact you, ideally Discord or email!

**Comment by Andres6936** [comment]
I can help you with the translations, however depending on the amount of words to translate, it will take some time, I am not committed to deliver in a day or two the translations.

**Reply by TiaPixel** [reply]
There isn't a deadline, but it would be good to know how long it might take.

The Ideology store page (https://store.steampowered.com/app/1392840/RimWorld__Ideology/) has 3703 words.
The Royalty store page (https://store.steampowered.com/app/1149640/RimWorld__Royalty/) has 1609 words.

Do you have a Discord or an email I can reach you at? :)

**Comment by Andres6936** [comment]
A quick initial translation can take between 3 and 4 days, and an accurate correction 1 additional week, likewise, if it would be convenient to be able to modify the translations some time after the initial translation in case there are variations of the translation within the game, so that the changes could be reflected correctly both from Steam and within the game.

My email: andres6936@live.com

**Comment by Zakees** [comment]
As this was already translated, I close this. Thanks for help! ^^
